.circleci/config.yml

```diff
@@ -132,9 +175,59 @@ jobs:
       - checkout
       - cmd_install-prerequisites
       - cmd_install-dependencies
-      - cmd_install-chrome-utils
+      - cmd_install-chrome-utils-linux
       - cmd_run-build
+      - cmd_set-linux-permissions
       - cmd_run-e2e-tests
+      - store_artifacts:
+          path: packages/flex-plugin-e2e-tests/screenshots
+
+  job_e2e-test-win:
+    parameters:
+      package_version:
+        type: string
+      shell:
+        type: string
+    environment:
+      NODE_OPTIONS: --max-old-space-size=14336
+      PACKAGE_VERSION: << parameters.package_version >>
+    executor:
+      name: win/default
+      shell: << parameters.shell >>
+      size: xlarge
+    steps:
+      - checkout
+      - cmd_windows-nvm-install-node:
+          node-version: "12.13.0"
+      - cmd_install-dependencies
+      - cmd_install-chrome-utils-win
+      - cmd_run-build
+      - cmd_run-e2e-tests
+      - store_artifacts:
```

Comment:
What does this do?

Reply:
It will store anything you provide in artifacts. If you go to https://app.circleci.com/pipelines/github/twilio/flex-plugin-builder/690/workflows/7e4867c6-3524-4797-8e32-5c0ae7327bf4/jobs/2900/steps and look under artifact tab, you will see that the screenshot is stored there.